build/steps-build-android.yml

```diff
@@ -0,0 +1,91 @@
+parameters:
+- name: androidKeyStoreFile
+  type: string
+  default: ''
+- name: pathToSrc
+  type: string
+  default: ''
+- name: solutionName
+  type: string
+  default: ''
+- name: pathToAndroidManifest
+  type: string
+  default: ''
+
+steps:
+- template: templates/dotnet-install.yml
```

Comment:
is it still necessary? I though we didn't need it

Reply:
for android & winui

Reply:
Yes it is, it makes sure that we have the correct version of Dot Net installed and that we have the necessary mobile workloads.